client/websocket_demo.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE HTML>
+<html>
+   <head>
+	
+      <script type="text/javascript">
+         function WebSocketTest()
+         {
+            if ("WebSocket" in window)
+            {
+               
+               // Let us open a web socket
+               var ws = new WebSocket("ws://localhost:8080/websocket");
```

Comment:
This has been fixed in commit c15cb6f. However this is not intended for use in prod or staging but instead a way to show the frontend developers how web sockets can be implemented.